Makefile

```diff
@@ -0,0 +1,52 @@
+artifact_name       := third-party-test-harness
+version             := unversioned
+
+.PHONY: all
+all: build
+
+.PHONY: clean
+clean:
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+	rm -f ./build.log
+
+.PHONY: build
+build:
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
+	cp ./target/$(artifact_name)-$(version).jar ./$(artifact_name).jar
+
+.PHONY: test
+test: test-unit
+
+.PHONY: test-unit
+test-unit: clean
+	mvn test
+
+.PHONY: package
+package:
+ifndef version
+	$(error No version given. Aborting)
+endif
+	$(info Packaging version: $(version))
+	mvn versions:set -DnewVersion=$(version) -DgenerateBackupPoms=false
+	mvn package -DskipTests=true
+	$(eval tmpdir:=$(shell mktemp -d build-XXXXXXXXXX))
+	cp ./start.sh $(tmpdir)
+	cp ./routes.yaml $(tmpdir)
+	cp ./target/$(artifact_name)-$(version).jar $(tmpdir)/$(artifact_name).jar
+	cd $(tmpdir); zip -r ../$(artifact_name)-$(version).zip *
+	rm -rf $(tmpdir)
+
+.PHONY: dist
+dist: clean build package
+
+.PHONY: sonar
+sonar:
+	mvn sonar:sonar
+#commented out until Sonar is added to project via pipeline
+#.PHONY: sonar-pr-analysis
+#sonar-pr-analysis:
+#	mvn sonar:sonar -P sonar-pr-analysis
```

Comment:
If we are not enabling this, I'd remove it altogether rather than comment it out. It can be added later on if needed

Reply:
I'll remove on next commit 👍